### PR TITLE
New recipe: amath 1.7.1

### PR DESCRIPTION
**Pull request by rainlance**
Tested and verified as described in GoogleCodeInTaskRecipe. Possible missing copyrights in recipe (see amath license).
snap1

### CODE REVIEW

**Comment by rainlance** [comment]
snap2

**Comment by scottmc** [comment]
Note: What taking a screenshot with screencapture there is an option to have it include the tab and make the rest transparent, that way the crop is done for you.

File: sci-mathematics/amath/amath-1.7.0.recipe [hunk 1]
```diff
@@ -0,0 +1,49 @@
+SUMMARY="Simple command line calculator"
+DESCRIPTION="amath features a case sensitive command line interface, internal
+IEEE 754 calculations with 15 significant digits, calculations with real and
+complex numbers, variables and user defined functions, logarithmic and exponential \
+functions, trigonometric and hyperbolic function and selected mathematical \
+constants and rounding functions."
+HOMEPAGE="http://amath.innolan.net"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="Carsten Sonne Larsen"
```
Comment (scottmc) [inline sci-mathematics/amath/amath-1.7.0.recipe]: Should be:
2014-2017 Carsten Sonne Larsen

Comment (rainlance) [inline sci-mathematics/amath/amath-1.7.0.recipe]: @scottmc How to use the option of having it included in the tab and make the rest transparent ? I would like to try. Could you give a link or some other guideline ?

**Comment by scottmc** [comment]
https://www.haiku-os.org/docs/userguide/en/applications/screenshot.html
You check the capture active window box, i think.  Play around with the settings, I know one of them does it.

**Comment by rainlance** [comment]
New version with Haiku specific GUI using BWindow, BTextView BAboutWindow, and with x11 console fixes when running in shell mode.

**Comment by waddlesplash** [comment]
> New version with Haiku specific GUI 

I don't see that..? Or does amath itself have one.

**Comment by rainlance** [comment]
Sorry. Had to make a new pull request. See #1245
Changes has not yet been not pushed to amath repo as I want to make sure the changes for 1.7.1 are approved. Most of them are related to Haiku.
Distribution site (SOURCE_URI) is the same as for other versions of amath.

**Pull request closed without merging**